rosdep/base.yaml

```diff
@@ -6447,6 +6450,7 @@ log4cxx:
   arch: [log4cxx]
   cygwin: [liblog4cxx-devel]
   debian:
```

Comment:
I can't add a suggestion here, but I think we should simplify this particular section.  The only distributions where this is called `liblog4cxx10-dev` are long end-of-life, so I think this whole thing can just be:

```
debian: [liblog4cxx-dev]
```